src/Tooltip/index.js

```diff
@@ -437,13 +442,16 @@ class Tooltip extends React.PureComponent<Props, State> {
           onFocus={this.handleIn}
           onBlur={this.handleOut}
           ref={this.container}
+          aria-describedby={this.tooltipId}
+          tabIndex="0"
         >
           {children}
         </StyledTooltipChildren>
         <Portal element="tooltips">
           <StyledTooltip role="tooltip" data-test={dataTest}>
```

Comment:
Tooltip has now `role` twice. Is in intentional?

Reply:
Good catch!